pkg/authentication/handlers.go

```diff
@@ -258,7 +258,11 @@ func (a *API) handleLogout(w http.ResponseWriter, r *http.Request) {
 }
 
 func (a *API) uiRedirect(w http.ResponseWriter, r *http.Request, nextTo string) {
-	redirect := ui.UIPrefix
+	redirect, err := url.JoinPath("/", a.contextPath, ui.UIPrefix)
+	if err != nil {
+		a.badRequest(w, err)
```

Comment:
If we ever end up here, it means that the user has misconfigured the app. I think we should log it and return `500` instead

Reply:
Done! Thanks